workflow/rocoto/gfs_tasks.py

```diff
@@ -18,6 +18,13 @@ def _is_this_a_gdas_task(run, task_name):
     # Specific Tasks begin here
     def stage_ic(self):
 
+        deps = []
+        dep_dict = {
+            'type': 'task', 'name': f'fetch',
+        }
+        deps.append(rocoto.add_dependency(dep_dict))
+        dependencies = rocoto.create_dependency(dep=deps)
+
```

Comment:
This should only be a dependency if the `fetch` job exists:

```suggestion
        if self.options['fetch_hpss'] or self.options['fetch_local']:
            deps = []
            dep_dict = {
                'type': 'task', 'name': f'fetch',
            }
            deps.append(rocoto.add_dependency(dep_dict))
            dependencies = rocoto.create_dependency(dep=deps)
```

Reply:
Updated those lines.